### ISSUE TITLE
accessibility

### ISSUE THREAD

**Issue by bridgetf7** [sitechtimes/SITHS-website-redesign#114]
Using accessibility checker tools, ADA, and WCAG compliance guidelines, ensure the website is fully accessible

**Comment by katiekhamarkhanova** [comment]
I used the WAVE extension to check if the website aligned with the WCAG compliance guidelines:
- The pages on the website are missing titles which can make it hard for users to understand the purpose or content of each page.
- The language (English) of the pages is not specified. 
- The "Continue Reading" link on the School History section on the main page has a low contrast error. The link text can also be a bit confusing out of context for those who use screen readers: 
![Image](https://github.com/user-attachments/assets/f63a51bf-2bc4-45a9-844f-e5b79d8a9854)

- "The Early Bird" text and "The Early Bird" alt text for the image on the home page come up as redundant. I think the alt text could be changed to specify it as an image. 
![Image](https://github.com/user-attachments/assets/1fc2c130-1e40-4e21-b178-f9c67b73655d)

- The Twitter logo has the wrong alt text. It says "Instagram Logo" 
![Image](https://github.com/user-attachments/assets/f1c77f29-cce1-4f40-97cd-2c9532e24d0b)

- The "Eleven Minute Homeroom" heading is an h3, which skipped a heading level from the previous heading. This could be confusing for screen readers
![Image](https://github.com/user-attachments/assets/2fe7aea9-7e00-46af-8150-24a6112d4436)

- The "Automated Telephone Message Directory" and "718-667-3222" heading also skip a heading level from the previous h1 into an h3.
![Image](https://github.com/user-attachments/assets/e7935e6b-44eb-4644-a238-f8ac9a62cb4c)

- On the CTE Partnerships page, the link for Jax's email and link to become a guest speaker are both invisible and blend in with the background so I only see them when I highlight the text
![Image](https://github.com/user-attachments/assets/084e52f0-3c69-4f82-bb73-cd20b358eb2d)
![Image](https://github.com/user-attachments/assets/22f2b17c-fd07-4b53-9c50-0fa19cce86b0)

- There is an empty heading on the Summer Homework page which can be confusing for keyboard and screen reader users
![Image](https://github.com/user-attachments/assets/36e5898e-0344-447c-b405-69664893ebdf)

- On Mr. Terrusa's Activity Updates, the COSA (which is an h4) skips a heading level from previous heading which was an h2 
![Image](https://github.com/user-attachments/assets/bc6031c0-3163-42fb-a7c3-81e2008e0ece)
-On Mr. Terrusa's Activity Updates, the image for the homecoming dance section has the alt text: "Post Image" which is misleading for screen readers
![Image](https://github.com/user-attachments/assets/0688e94d-d96d-4bcd-8c8c-2da2b078e3c6)
- On Mr. Terrusa's Activity Updates, the form for the drop down does not have a label? I don't know if it's supposed to
![Image](https://github.com/user-attachments/assets/569b8d52-3f80-4eba-92ee-e7bcf472359b)

- On the School Year Information page, the alt text for the 2024-2025 School Year Info image is "post image" which can be misleading for screen readers
![Image](https://github.com/user-attachments/assets/a4119935-e51b-4d36-9286-c7a4fad95392)

- On the School Year Information and Principal Erlenwein's Announcements pages, the heading for "Principal" which is an h4 skips a level from the previous h2 "Mark Erlenwein" 
![Image](https://github.com/user-attachments/assets/f7eee95e-8522-4598-b63a-44fe538cb35c)
- On the School Year Information and Principal Erlenwein's Announcements pages, the form for the dropdowns do not have proper labels for their functions
![Image](https://github.com/user-attachments/assets/6a04f856-484a-4b02-a139-09a11f335451)

- All of headings after the h1 titled "Resources" are h4s which skip heading levels.
![Image](https://github.com/user-attachments/assets/df47aea9-9505-4069-9bfd-e4486aab0141)

- On the Athletics About Page, the heading for "McKee - Staten Island Tech Public Schools Athletics League" which is an h3 skips a level from the previous h1
![Image](https://github.com/user-attachments/assets/fea04b3d-73f8-4ece-b662-08a058056b11)

-On the Athletics Contacts Page, the heading for "All (A-Z)" is an h3 and skips a level from the previous h1. In addition, the search form does not have a corresponding label which can make it difficult for screen readers to decipher the purpose of the form
![Image](https://github.com/user-attachments/assets/4c9a8e07-9d8b-4c90-8cf0-479bf77b81d8)

- On the Special Opportunities page, the cards on the Internships, Scholarships, Volunteer, and Initiative sections all have headings that skip a level from the h2 at the top of each section (they're all h4)
![Image](https://github.com/user-attachments/assets/2aeda2ed-b2b6-4b0d-90b6-26a0248c2d20)
![Image](https://github.com/user-attachments/assets/ce43004e-d3e2-4b47-8c52-87bcf1cb8cbc)
![Image](https://github.com/user-attachments/assets/3c544757-dd95-4b10-acd0-eaa8005b540f)
![Image](https://github.com/user-attachments/assets/593c7e45-1f06-49f6-9a5e-421cc410d367)

**Comment by katiekhamarkhanova** [comment]
- On the Error 404 Page, the text "Drawing by Yifan Zhang '25" has a low contrast with the background
![Image](https://github.com/user-attachments/assets/c7c9eabf-a6ed-41cb-be91-d2c04c7b086e)
